src/route/RootRouter.tsx

```diff
@@ -33,6 +34,9 @@ const RootRouter = (): JSX.Element => {
         <Route path="/mypetitions" element={<AuthRoute />}>
           <Route index element={<MyPetitions />} />
         </Route>
+        <Route path="/answered" element={<Outlet />}>
```

Comment:
```suggestion
        <Route path="/answer" element={<Outlet />}>
```

일반 동사, 명사만을 사용하게끔 path명만 바꿔줍시다!